add Zhang-2019-308 concept list

MacyL opened the pull request:
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

LinguList commented:
Okay, thanks, @MacyL, I now invited @Schweikhard, @Kristina-Pianykh and @CarolinHu to review this. Note that this should not take too long a time. If you are unsure about Chinese concepts, I'll check with @MacyL.

LinguList commented on concepticondata/conceptlists.tsv:
```diff
@@ -291,13 +291,13 @@ Schroeder-2012-824	Schroeder, Astrid and Gemballa, Teresa and Ruppin, Steffie an
 Xiao-2012-213	Xiao, Wen and Treiman, Rebecca	2012		213	ranked	Chinese	Chinese	https://link.springer.com/article/10.3758/s13428-012-0191-3#SecESM1	Xiao2012		40 U.S. adults with no knowledge of Chinese were given an English word or short phrase together with two Chinese characters and were asked which character matched the meaning of the English word. 213 simple-structure Chinese characters that oc ...
 Desrochers-2010-330	Desrochers, Alain and Liceras, Juana M. and Fernandez-Fuertes, Raquel and Thompson, Glenn L.	2010		330	ranked	Spanish	Spanish	https://link.springer.com/article/10.3758/BRM.42.1.109#SecESM1	Desrochers2010		Subjective frequency ratings from 102 Spanish native speakers of 330 words (compounds, nouns and verbs). Objective frequency norms and mean subjective familiarity ratings wer ...
-Monnier-2014-1031	Monnier, Catherine and Syssau, Arielle	2014		1031	ranked	French	French	https://link.springer.com/article/10.3758/s13428-013-0431-1#SupplementaryMaterial	Monnier2014		The stimulus set contained 1,031 French words (969 nouns and 62 adjectives). The words in FAN cover a wide range of categories (objects, animals, food, and concepts). Ratings were made using the Self-Assessment Mani ...
+Monnier-2014-1031	Monnier, Catherine and Syssau, Arielle	2014		1031	ranked	French	French	https://link.springer.com/article/10.3758/s13428-013-0431-1#SupplementaryMaterial	Monnier2014		The stimulus set contained 1,031 French words (969 nouns and 62 adjectives). The words in FAN cover a wide range of categories (objects, animals, food, and concepts). Ratings were made using the Self-Assessment Mani ...
```
@MacyL, please could you check the difference between these lines? Is it additional tabstops, or missing ones? It is important to check these diffs.

MacyL replied:
I guess it's Mac numbers added some quotation marks during export, I will check the file and fix it.

LinguList commented on concepticondata/conceptlists.tsv:
```diff
@@ -310,10 +310,11 @@ Zettersten-2020-16b	Zettersten, Martin and Lupyan, Gary	2020	b	16	naming test	En
 Ruhlen-1998-36	Ruhlen, Merritt	1998		36	basic, areal	English	Yeniseian and Na-Dene	https://www.pnas.org/content/pnas/95/23/13994.full.pdf	Ruhlen1998		The basic concepts in this list serve to compare Proto-Yeniseian and Proto-Na-Dene cognates which are not widely shared by other language families. The author argues that this large amount of cognates of basic vocabulary can only be explained by a c ...
 Lee-2015-246	Lee, Sean	2015		246	basic, areal	English	Koreanic	https://doi.org/10.1371/journal.pone.0128448	Lee2015		The basic concepts in this list are selected to &quot;quantitatively describe the evolutionary characteristics&quot; (of Koreanic).	e0128448	
 Urban-2016-160	Urban, Matthias	2016		160	basic	English	Global		Urban2016	Urban2016	This list of 160 nominal items was compiled in order to compare morphological complexity across 78 languages of different language families. The concepts were organized into four semantic categories relating to nature, artifacts, the body and temporal concepts (plus some miscellaneous words). The list is not direct ...
+Zhang-2019-308	Zhang, Shuya and Jacques, Guillaume and Lai, Yunfan	2019		308	basic	English	rGyalrong and Old Chinese	https://pure.mpg.de/pubman/faces/ViewItemOverviewPage.jsp?itemId=item_3149499	Zhang2019		This list of 122 items was extracted from a study on cognates between rGyalrong languages and Old Chinese.	73-92	
```
The list now has 308 items, right?

MacyL replied:
yes, it has 308 items

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-30	30	the child	child	2099	CHILD	
+Zhang-2019-308-31	31	the claw	claw	72	CLAW	
+Zhang-2019-308-32	32	the clothes	clothes	1895	CLOTHES	
+Zhang-2019-308-33	33	the colour, countenance	colour, countenance	1613	COLOR	
```
Hm, countenance means a face or a facial expression, doesn't it? I wouldn't map it then to COLOR.

laiyunfan replied:
The word is 色 'colour'. I think it already primarily meant colour in Old Chinese.

Schweikhard commented:
Hm, is this supposed to be based on the original English now? Or the modified English? Or the Chinese? Because there is no Chinese column. And some mappings differ a bit from the original English at least (e.g. I wouldn't map "countenance colour, countenance" to COLOR).

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-34	34	the command	command	1998	COMMAND	
+Zhang-2019-308-35	35	the cooked rice	cooked rice	1191	COOKED RICE	
+Zhang-2019-308-36	36	the cow	cow	1007	COW	
+Zhang-2019-308-37	37	to cry (of birds or animals)	cry (of birds or animals)	1839	CRY	
```
CRY 1839 refers to the act of shedding tearing under the influence of emotions. This is clearly a different concept.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-45	45	eight	eight	1705	EIGHT	
+Zhang-2019-308-46	46	the elbow	elbow	981	ELBOW	
+Zhang-2019-308-47	47	be eldest, great	eldest, great			concept only existent in Old Chinese
+Zhang-2019-308-48	48	be evil	evil	3764	BAD OR EVIL	
```
Maybe 45 EVIL?

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-64	64	be fresh	fresh	243	FRESH	
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
```
Maybe leave it unmapped given there is another CLOTHES instance on the list?

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
+Zhang-2019-308-68	68	the grandchild, nephew	grandchild, nephew	1617	GRANDCHILD	
```
I would unlink this one too because GRANDCHILD would be too narrow for 'the grandchild, nephew'.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-81	81	the ice	ice	617	ICE	
+Zhang-2019-308-82	82	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-83	83	the internal organs	internal organs	1334	GUTS	
+Zhang-2019-308-84	84	the intestine	intestine	1991	INTESTINES	
```
Could you check with the source once again if this duplicate is intended to be here? (see Zhang-2019-308-82)

MacyL replied:
@laiyunfan

Schweikhard commented:
Ah, didn't scroll down that far yet, and didn't expect that the English glosses and the Old Chinese would be included in one and the same list, I thought we'd now put them in different lists or only use the Chinese.
In that case, that certainly needs to be explained in the list description.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-82	82	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-83	83	the internal organs	internal organs	1334	GUTS	
+Zhang-2019-308-84	84	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-85	85	the jaw, chin	jaw, chin	798	JAW	
```
In Nagano-2013-1256-37 'jaw, chin' is mapped to CHIN but I don't know what lexemes are used here and in Nagano's list.

MacyL replied:
I agree, I think CHIN is a better option, but @laiyunfan could you have a look?

laiyunfan replied:
Frankly, I don't know the exact differences between chin and jaw in this case. But if Nagano has CHIN, let it be CHIN.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-88	88	the louse	louse	1392	LOUSE	
+Zhang-2019-308-89	89	the milk, nipple	milk, nipple	2129	BREAST OR MILK	
+Zhang-2019-308-90	90	the millet	millet	931	MILLET	
+Zhang-2019-308-91	91	the millet or rice grains	millet or rice grains	931	MILLET	
```
Maybe unmap this one given MILLET above?

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-108	108	the plant root	plant root	670	ROOT	
+Zhang-2019-308-109	109	the pliers	pliers			
+Zhang-2019-308-110	110	the poison	poison	655	POISON	
+Zhang-2019-308-111	111	the pulse, beans	pulse, beans	832	BEAN	
```
Wikipedia says pulses can refer to various kinds of seeds, including beans, peas, lentils, etc. So I think this case calls for a broader concept set.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-133	133	the tail	tail	1220	TAIL	
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
```
Is the specification (OF SHAPE OF OBJECT) justified by the meaning of the target lexemes? If yes, then ignore my comment.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
+Zhang-2019-308-137	137	thou	thou	1215	THOU	
+Zhang-2019-308-138	138	three	three	492	THREE	
+Zhang-2019-308-139	139	to beat	to beat	2133	STRIKE OR BEAT	
```
Maybe BEAT 1665?

MacyL replied:
@laiyunfan

laiyunfan replied:
I would opt for the more general STRIKE OR BEAT...

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-139	139	to beat	to beat	2133	STRIKE OR BEAT	
+Zhang-2019-308-140	140	to bend	to bend	2224	BEND (SOMETHING)	
+Zhang-2019-308-141	141	to break (vi)	to break (vi)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-142	142	to break (vt)	to break (vt)	680	BREAK (DESTROY OR GET DESTROYED)	
```
See above, is this the intentional use of duplicates?

laiyunfan replied:
One is transitive and the other is intransitive, the concept implies both.

Schweikhard replied:
Then the first should be mapped to BREAK (BREAKING), the second to BREAK (CLEAVE).

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-144	144	to chase	to chase	1047	PURSUE	
+Zhang-2019-308-145	145	to chew	to chew	321	CHEW	
+Zhang-2019-308-146	146	to confer	to confer			
+Zhang-2019-308-147	147	to curve	to curve	2224	BEND (SOMETHING)	
```
I'd leave this one unmapped because of 'bend' above.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-150	150	to destroy	to destroy	681	DESTROY	
+Zhang-2019-308-151	151	to die	to die	1494	DIE	
+Zhang-2019-308-152	152	to dig out (earth)	to dig out (earth)	1418	DIG	
+Zhang-2019-308-153	153	to drop, drop	to drop, drop	808	DRIP (FALL IN GLOBULES)	
```
This must be specifically expressed through the Chinese lexemes but I can't judge by the English gloss if it should be mapped to DRIP (FALL IN GLOBULES)

MacyL replied:
@laiyunfan

laiyunfan replied:
After checking the data, I confirm the current mapping.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-153	153	to drop, drop	to drop, drop	808	DRIP (FALL IN GLOBULES)	
+Zhang-2019-308-154	154	to extinguish	to extinguish	127	EXTINGUISH	
+Zhang-2019-308-155	155	to fall	to fall	1280	FALL	
+Zhang-2019-308-156	156	to fall (from hand)	to fall (from hand)	1280	FALL	
```
Unmap to avoid duplication?

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-129	129	to strike	strike	353	STRIKE	
+Zhang-2019-308-130	130	the string, cord	string, cord	362	STRING	
+Zhang-2019-308-131	131	be strong, hard	strong, hard	1884	HARD	
+Zhang-2019-308-132	132	to suffer defeat	suffer defeat	1660	LOSE	
```
Maybe better 3535 BE DEFEATED? LOSE then can be reserved for Zhang-2019-308-162

laiyunfan replied:
Good idea.

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-164	164	to peck	to peck	3684	PECK	
+Zhang-2019-308-165	165	to plant	to plant	1486	PLANT (SOMETHING)	
+Zhang-2019-308-166	166	to press	to press	1913	PRESS	
+Zhang-2019-308-167	167	to revive	to revive	1602	WAKE UP	
```
WAKE UP seems too specific, judged by the English gloss only. 'revive' refers also to gaining consciousness, I suppose.

LinguList commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-22	22	the belly	belly	1251	BELLY	
+Zhang-2019-308-23	23	the boat	boat	1844	BOAT	
+Zhang-2019-308-24	24	the body, person	body, person	1480	BODY	
+Zhang-2019-308-25	25	the body, self	body, self	1480	BODY	
```
This is fine in this context, @CarolinHu, as this reflects the original data, and glosses can in fact be repeated.

LinguList commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-164	164	to peck	to peck	3684	PECK	
+Zhang-2019-308-165	165	to plant	to plant	1486	PLANT (SOMETHING)	
+Zhang-2019-308-166	166	to press	to press	1913	PRESS	
+Zhang-2019-308-167	167	to revive	to revive	1602	WAKE UP	
```
There is no original Chinese here. So I'd say: unlink, @MacyL

LinguList commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-189	189	seven	七	1704	SEVEN	
+Zhang-2019-308-190	190	three	三	492	THREE	
+Zhang-2019-308-191	191	descend	下	2014	GO DOWN (DESCEND)	
+Zhang-2019-308-192	192	you r	乃	732	YOUR (SINGULAR)	
```
Yesm @MacyL, just correct the typo here.

LinguList commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-305	305	the horse	馬	615	HORSE	
+Zhang-2019-308-306	306	the thighbone	髀			
+Zhang-2019-308-307	307	be fresh	鮮	243	FRESH	
+Zhang-2019-308-308	308	to cry (of birds or animals)	鳴	1839	CRY	
```
we don't have the concept set for bird calls, but this is what this originally seems to have meant.

laiyunfan replied:
鳴 is also attested for deer calls: 呦呦鹿鳴，食野之萍, but I don't know how restricted its uses were in Old Chinese.

LinguList commented:
Yes, @MacyL, the description needs to say: 

> Since the data is a collection of etymological items with very divergent meanings and elicitation glosses, we assembled the data for each language separately and added a unified layer of translation. 

@MacyL, note also, that you should modify the ENGLISH column name to GLOSS, this is our standard column name for custom translations to English that we do which are not in the original data.

MacyL commented:
Check: unique_concepticon_gloss
|CONCEPTICON_ID   |   LINE_NO  |  ID                    | NUMBER  | GLOSS|
|--------------------|-----------| -----------------| ---------| --------|
|                    |2        |Zhang-2019-308-1        |1        |diminutive suffix|
|        |7        |Zhang-2019-308-6        |6        |the Allium tuberosum|
|        |39        |Zhang-2019-308-38        |38        |the dark side of the mountain|
|        |40        |Zhang-2019-308-39        |39        |the deceased mother|
|        |48        |Zhang-2019-308-47        |47        |be eldest, great|
|        |54        |Zhang-2019-308-53        |53        |the farming|
|        |58        |Zhang-2019-308-57        |57        |the father's sister's child|
|        |67        |Zhang-2019-308-66        |66        |the fur garment|
|        |110        |Zhang-2019-308-109        |109        |the pliers|
|        |125        |Zhang-2019-308-124        |124        |the snake brood|
|        |129        |Zhang-2019-308-128        |128        |be stiff|
|        |135        |Zhang-2019-308-134        |134        |there is no X such that X...|
|        |144        |Zhang-2019-308-143        |143        |to carry water on back|
|        |147        |Zhang-2019-308-146        |146        |to confer|
|        |169        |Zhang-2019-308-168        |168        |to scoop water|
|        |218        |Zhang-2019-308-217        |217        |the dead mother|
|        |220        |Zhang-2019-308-219        |219        |be eldest, greatest|
|        |244        |Zhang-2019-308-243        |243        |to scoop water|
|        |257        |Zhang-2019-308-256        |256        |be stiff|
|        |282        |Zhang-2019-308-281        |281        |there is no X such that X...|
|        |287        |Zhang-2019-308-286        |286        |the snake brood|
|        |291        |Zhang-2019-308-290        |290        |the fur garment|
|        |303        |Zhang-2019-308-302        |302        |the Allium tuberosum|
|        |307        |Zhang-2019-308-306        |306        |the thighbone|
|10        |61        |Zhang-2019-308-60        |60        |the firewood|
|10        |285        |Zhang-2019-308-284        |284        |the firewood|
|1044        |108        |Zhang-2019-308-107        |107        |the pillow|
|1044        |238        |Zhang-2019-308-237        |237        |pillow|
|1173        |52        |Zhang-2019-308-51        |51        |the eyebrow|
|1173        |261        |Zhang-2019-308-260        |260        |the eyebrow|
|1209        |4        |Zhang-2019-308-3        |3        |I|
|1209        |210        |Zhang-2019-308-209        |209        |I|
|1217        |56        |Zhang-2019-308-55        |55        |the father|
|1217        |253        |Zhang-2019-308-252        |252        |the father|
|1220        |134        |Zhang-2019-308-133        |133        |the tail|
|1220        |222        |Zhang-2019-308-221        |221        |the tail|
|1231        |18        |Zhang-2019-308-17        |17        |new|
|1231        |234        |Zhang-2019-308-233        |233        |be new|
|1233        |99        |Zhang-2019-308-98        |98        |the night|
|1233        |215        |Zhang-2019-308-214        |214        |the night|
|1247        |44        |Zhang-2019-308-43        |43        |the ear|
|1247        |271        |Zhang-2019-308-270        |270        |the ear|
|1248        |51        |Zhang-2019-308-50        |50        |the eye|
|1248        |260        |Zhang-2019-308-259        |259        |the eye|
|1251        |23        |Zhang-2019-308-22        |22        |the belly|
|1251        |278        |Zhang-2019-308-277        |277        |the belly|
|1278        |186        |Zhang-2019-308-185        |185        |to vomit|
|1278        |213        |Zhang-2019-308-212        |212        |to vomit|
|1280        |156        |Zhang-2019-308-155        |155        |to fall|
|1280        |157        |Zhang-2019-308-156        |156        |to fall (from hand)|
|1306        |128        |Zhang-2019-308-127        |127        |the sovereign, queen|
|1306        |209        |Zhang-2019-308-208        |208        |the sovereign, queen|
|1324        |174        |Zhang-2019-308-173        |173        |to slip|
|1324        |294        |Zhang-2019-308-293        |293        |to slip|
|133        |181        |Zhang-2019-308-180        |180        |to weave|
|133        |267        |Zhang-2019-308-266        |266        |to weave|
|1331        |119        |Zhang-2019-308-118        |118        |the sheep|
|1331        |270        |Zhang-2019-308-269        |269        |the sheep|
|1334        |84        |Zhang-2019-308-83        |83        |the internal organs|
|1334        |276        |Zhang-2019-308-275        |275        |the internal organs|
|1382        |98        |Zhang-2019-308-97        |97        |the needle|
|1382        |299        |Zhang-2019-308-298        |298        |the needle|
|1392        |89        |Zhang-2019-308-88        |88        |the louse|
|1392        |289        |Zhang-2019-308-288        |288        |the louse|
|1393        |78        |Zhang-2019-308-77        |77        |the horn|
|1393        |292        |Zhang-2019-308-291        |291        |the horn|
|1405        |96        |Zhang-2019-308-95        |95        |the name|
|1405        |208        |Zhang-2019-308-207        |207        |the name|
|1416        |173        |Zhang-2019-308-172        |172        |to sit|
|1416        |214        |Zhang-2019-308-213        |213        |to sit|
|1417        |162        |Zhang-2019-308-161        |161        |to kill|
|1417        |241        |Zhang-2019-308-240        |240        |to kill|
|1418        |153        |Zhang-2019-308-152        |152        |to dig out (earth)|
|1418        |229        |Zhang-2019-308-228        |228        |to dig out (earth)|
|1442        |176        |Zhang-2019-308-175        |175        |to stand|
|1442        |263        |Zhang-2019-308-262        |262        |to stand|
|1447        |159        |Zhang-2019-308-158        |158        |to give|
|1447        |256        |Zhang-2019-308-255        |255        |to give|
|148        |161        |Zhang-2019-308-160        |160        |to itch|
|148        |258        |Zhang-2019-308-257        |257        |to itch|
|1480        |25        |Zhang-2019-308-24        |24        |the body, person|
|1480        |26        |Zhang-2019-308-25        |25        |the body, self|
|1480        |295        |Zhang-2019-308-294        |294        |the body|
|1480        |296        |Zhang-2019-308-295        |295        |the body|
|1483        |100        |Zhang-2019-308-99        |99        |nine|
|1483        |194        |Zhang-2019-308-193        |193        |nine|
|1493        |102        |Zhang-2019-308-101        |101        |one|
|1493        |123        |Zhang-2019-308-122        |122        |be single|
|1493        |301        |Zhang-2019-308-300        |300        |be single|
|1494        |152        |Zhang-2019-308-151        |151        |to die|
|1494        |240        |Zhang-2019-308-239        |239        |to die|
|1498        |184        |Zhang-2019-308-183        |183        |two|
|1498        |196        |Zhang-2019-308-195        |195        |two|
|1504        |64        |Zhang-2019-308-63        |63        |the fly|
|1504        |290        |Zhang-2019-308-289        |289        |the fly (insect)|
|1560        |53        |Zhang-2019-308-52        |52        |the face|
|1560        |305        |Zhang-2019-308-304        |304        |the face|
|1574        |63        |Zhang-2019-308-62        |62        |to float|
|1574        |245        |Zhang-2019-308-244        |244        |to float|
|1583        |27        |Zhang-2019-308-26        |26        |the brain|
|1583        |277        |Zhang-2019-308-276        |276        |the brain|
|1602        |168        |Zhang-2019-308-167        |167        |to revive|
|1602        |286        |Zhang-2019-308-285        |285        |to revive|
|1613        |34        |Zhang-2019-308-33        |33        |the colour, countenance|
|1613        |281        |Zhang-2019-308-280        |280        |the colour, countenance|
|1634        |80        |Zhang-2019-308-79        |79        |hundred|
|1634        |259        |Zhang-2019-308-258        |258        |hundred|
|1660        |133        |Zhang-2019-308-132        |132        |to suffer defeat|
|1660        |163        |Zhang-2019-308-162        |162        |to lose|
|1660        |217        |Zhang-2019-308-216        |216        |to lose|
|1685        |5        |Zhang-2019-308-4        |4        |after|
|1685        |226        |Zhang-2019-308-225        |225        |after|
|1689        |175        |Zhang-2019-308-174        |174        |to smoke|
|1689        |250        |Zhang-2019-308-249        |249        |to smoke|
|1698        |13        |Zhang-2019-308-12        |12        |to be curved|
|1698        |223        |Zhang-2019-308-222        |222        |be curved|
|1703        |124        |Zhang-2019-308-123        |123        |six|
|1703        |202        |Zhang-2019-308-201        |201        |six|
|1704        |117        |Zhang-2019-308-116        |116        |seven|
|1704        |190        |Zhang-2019-308-189        |189        |seven|
|1705        |46        |Zhang-2019-308-45        |45        |eight|
|1705        |201        |Zhang-2019-308-200        |200        |eight|
|172        |122        |Zhang-2019-308-121        |121        |the shoulder blade|
|172        |279        |Zhang-2019-308-278        |278        |the shoulder blade|
|1767        |170        |Zhang-2019-308-169        |169        |to scrape|
|1767        |204        |Zhang-2019-308-203        |203        |to scrape|
|1820        |3        |Zhang-2019-308-2        |2        |the brown bear|
|1820        |21        |Zhang-2019-308-20        |20        |the bear|
|1820        |29        |Zhang-2019-308-28        |28        |the brown-and-white bear|
|1820        |249        |Zhang-2019-308-248        |248        |the bear|
|1820        |269        |Zhang-2019-308-268        |268        |the bear|
|1839        |38        |Zhang-2019-308-37        |37        |to cry (of birds or animals)|
|1839        |309        |Zhang-2019-308-308        |308        |to cry (of birds or animals)|
|1884        |16        |Zhang-2019-308-15        |15        |hard|
|1884        |132        |Zhang-2019-308-131        |131        |be strong, hard|
|1884        |205        |Zhang-2019-308-204        |204        |be hard|
|1886        |9        |Zhang-2019-308-8        |8        |the armpit|
|1886        |273        |Zhang-2019-308-272        |272        |the armpit|
|1886        |275        |Zhang-2019-308-274        |274        |the armpit|
|1895        |33        |Zhang-2019-308-32        |32        |the clothes|
|1895        |68        |Zhang-2019-308-67        |67        |the garment|
|1906        |127        |Zhang-2019-308-126        |126        |be sour|
|1906        |298        |Zhang-2019-308-297        |297        |be sour|
|1913        |167        |Zhang-2019-308-166        |166        |to press|
|1913        |293        |Zhang-2019-308-292        |292        |to press|
|1942        |97        |Zhang-2019-308-96        |96        |be near|
|1942        |297        |Zhang-2019-308-296        |296        |be near|
|1991        |83        |Zhang-2019-308-82        |82        |the intestine|
|1991        |85        |Zhang-2019-308-84        |84        |the intestine|
|1998        |35        |Zhang-2019-308-34        |34        |the command|
|1998        |207        |Zhang-2019-308-206        |206        |to command|
|2014        |150        |Zhang-2019-308-149        |149        |to descend|
|2014        |192        |Zhang-2019-308-191        |191        |descend|
|2099        |31        |Zhang-2019-308-30        |30        |the child|
|2099        |199        |Zhang-2019-308-198        |198        |the child|
|212        |59        |Zhang-2019-308-58        |58        |the field|
|212        |255        |Zhang-2019-308-254        |254        |the field|
|2146        |158        |Zhang-2019-308-157        |157        |to fry, roast|
|2146        |251        |Zhang-2019-308-250        |250        |to fry, roast|
|221        |60        |Zhang-2019-308-59        |59        |the fire|
|221        |248        |Zhang-2019-308-247        |247        |the fire|
|2224        |141        |Zhang-2019-308-140        |140        |to bend|
|2224        |148        |Zhang-2019-308-147        |147        |to curve|
|2224        |237        |Zhang-2019-308-236        |236        |to bend|
|2249        |137        |Zhang-2019-308-136        |136        |be thin|
|2249        |284        |Zhang-2019-308-283        |283        |be thin|
|2271        |179        |Zhang-2019-308-178        |178        |to think|
|2271        |227        |Zhang-2019-308-226        |226        |to think|
|2297        |43        |Zhang-2019-308-42        |42        |be dusk, dark|
|2297        |235        |Zhang-2019-308-234        |234        |be dusk, dark|
|2341        |15        |Zhang-2019-308-14        |14        |cloudy|
|2341        |300        |Zhang-2019-308-299        |299        |be dark, cloudy|
|2345        |113        |Zhang-2019-308-112        |112        |the rabbit, hare|
|2345        |200        |Zhang-2019-308-199        |199        |the rabbit, hare|
|2350        |45        |Zhang-2019-308-44        |44        |the earthworm|
|2350        |288        |Zhang-2019-308-287        |287        |the earthworm|
|2366        |103        |Zhang-2019-308-102        |102        |the onion|
|2366        |187        |Zhang-2019-308-186        |186        |the wild onion|
|2374        |41        |Zhang-2019-308-40        |40        |the dream|
|2374        |216        |Zhang-2019-308-215        |215        |the dream|
|243        |65        |Zhang-2019-308-64        |64        |be fresh|
|243        |308        |Zhang-2019-308-307        |307        |be fresh|
|2883        |164        |Zhang-2019-308-163        |163        |to oppose, disturb|
|2883        |231        |Zhang-2019-308-230        |230        |to oppose, disturb|
|2900        |177        |Zhang-2019-308-176        |176        |to take care of, protect|
|2900        |198        |Zhang-2019-308-197        |197        |to protect|
|321        |146        |Zhang-2019-308-145        |145        |to chew|
|321        |211        |Zhang-2019-308-210        |210        |to chew|
|3326        |75        |Zhang-2019-308-74        |74        |the hide|
|3326        |302        |Zhang-2019-308-301        |301        |the hide|
|353        |130        |Zhang-2019-308-129        |129        |to strike|
|353        |230        |Zhang-2019-308-229        |229        |to strike|
|3538        |178        |Zhang-2019-308-177        |177        |to take off|
|3538        |274        |Zhang-2019-308-273        |273        |to take off|
|362        |131        |Zhang-2019-308-130        |130        |the string, cord|
|362        |268        |Zhang-2019-308-267        |267        |the string, cord|
|3684        |165        |Zhang-2019-308-164        |164        |to peck|
|3684        |212        |Zhang-2019-308-211        |211        |to peck|
|3764        |49        |Zhang-2019-308-48        |48        |be evil|
|3764        |228        |Zhang-2019-308-227        |227        |be evil|
|425        |88        |Zhang-2019-308-87        |87        |be late|
|425        |236        |Zhang-2019-308-235        |235        |be late|
|430        |183        |Zhang-2019-308-182        |182        |to twist, to spin|
|430        |266        |Zhang-2019-308-265        |265        |to twist, to spin|
|492        |139        |Zhang-2019-308-138        |138        |three|
|492        |191        |Zhang-2019-308-190        |190        |three|
|493        |62        |Zhang-2019-308-61        |61        |five|
|493        |197        |Zhang-2019-308-196        |196        |five|
|598        |120        |Zhang-2019-308-119        |119        |the shell|
|598        |242        |Zhang-2019-308-241        |241        |the shell|
|615        |79        |Zhang-2019-308-78        |78        |the horse|
|615        |306        |Zhang-2019-308-305        |305        |the horse|
|617        |82        |Zhang-2019-308-81        |81        |the ice|
|617        |203        |Zhang-2019-308-202        |202        |the ice|
|655        |111        |Zhang-2019-308-110        |110        |the poison|
|655        |243        |Zhang-2019-308-242        |242        |the poison|
|670        |109        |Zhang-2019-308-108        |108        |the plant root|
|670        |206        |Zhang-2019-308-205        |205        |the root|
|675        |50        |Zhang-2019-308-49        |49        |the excrement|
|675        |224        |Zhang-2019-308-223        |223        |the excrement|
|677        |11        |Zhang-2019-308-10        |10        |the axe|
|677        |233        |Zhang-2019-308-232        |232        |the axe|
|680        |142        |Zhang-2019-308-141        |141        |to break (vi)|
|680        |143        |Zhang-2019-308-142        |142        |to break (vt)|
|681        |151        |Zhang-2019-308-150        |150        |to destroy|
|681        |246        |Zhang-2019-308-245        |245        |to destroy|
|712        |114        |Zhang-2019-308-113        |113        |be rich|
|712        |221        |Zhang-2019-308-220        |220        |be rich|
|72        |32        |Zhang-2019-308-31        |31        |the claw|
|72        |252        |Zhang-2019-308-251        |251        |the claw|
|732        |189        |Zhang-2019-308-188        |188        |your|
|732        |193        |Zhang-2019-308-192        |192        |you r|
|735        |104        |Zhang-2019-308-103        |103        |the owl|
|735        |239        |Zhang-2019-308-238        |238        |the owl|
|771        |72        |Zhang-2019-308-71        |71        |the hat|
|771        |225        |Zhang-2019-308-224        |224        |the hat|
|782        |149        |Zhang-2019-308-148        |148        |to defeat|
|782        |232        |Zhang-2019-308-231        |231        |to defeat|
|798        |86        |Zhang-2019-308-85        |85        |the jaw, chin|
|798        |304        |Zhang-2019-308-303        |303        |the jaw, chin|
|808        |154        |Zhang-2019-308-153        |153        |to drop, drop|
|808        |247        |Zhang-2019-308-246        |246        |to drop, drop|
|832        |112        |Zhang-2019-308-111        |111        |the pulse, beans|
|832        |283        |Zhang-2019-308-282        |282        |the pulse, beans|
|931        |91        |Zhang-2019-308-90        |90        |the millet|
|931        |92        |Zhang-2019-308-91        |91        |the millet or rice grains|
|977        |10        |Zhang-2019-308-9        |9        |the arrow|
|977        |262        |Zhang-2019-308-261        |261        |the arrow|
|981        |47        |Zhang-2019-308-46        |46        |the elbow|
|981        |272        |Zhang-2019-308-271        |271        |the elbow|
|999        |87        |Zhang-2019-308-86        |86        |the joint|
|999        |264        |Zhang-2019-308-263        |263        |the joint|


Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK

MacyL commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+ID	NUMBER	ENGLISH	GLOSS_IN_SOURCE	CONCEPTICON_ID	CONCEPTICON_GLOSS	REMARK
+Zhang-2019-308-1	1	diminutive suffix	(diminutive suffix)			
+Zhang-2019-308-2	2	the brown bear	Asiatic brown bear	1820	BEAR	"This word could generally mean ""bear"", as there are not many types of bears to the local people"
```
no, it was added because of Mac numbers. I will remove it, thanks :)

laiyunfan replied:
This is a multilingual dataset. All the concepts of all the languages in it are listed linearly in one column. So there are duplicate concepts.

MacyL commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-88	88	the louse	louse	1392	LOUSE	
+Zhang-2019-308-89	89	the milk, nipple	milk, nipple	2129	BREAST OR MILK	
+Zhang-2019-308-90	90	the millet	millet	931	MILLET	
+Zhang-2019-308-91	91	the millet or rice grains	millet or rice grains	931	MILLET	
```
I agree.

MacyL commented:
> Hm, is this supposed to be based on the original English now? Or the modified English? Or the Chinese? Because there is no Chinese column. And some mappings differ a bit from the original English at least (e.g. I wouldn't map "countenance colour, countenance" to COLOR).

The GLOSS_IN_SOURCE column contains English and Chinese. Chinese should be the glosses for the Old Chinese, and English is the glosses for modern languages

LinguList commented:
Yes, this is essential, @laiyunfan: the dataset gives a concepticon
concept set for **every word**, and when words have different
elicitation glosses, they may still be linked to the same concepticon
concept set, since there was no initial construction as a wordlist. So
duplicates are to be expected, and this can be mentioned in the note of
the file conceptlists.tsv.

laiyunfan commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
+Zhang-2019-308-68	68	the grandchild, nephew	grandchild, nephew	1617	GRANDCHILD	
```
It is OK to unmap. I checked the dataset, Tibetan colexifies granchild and nephew.

MacyL commented:
I pin myself @MacyL  to remind me to add this information to conceptlists.tsv
> Yes, this is essential, @laiyunfan: the dataset gives a concepticon concept set for **every word**, and when words have different elicitation glosses, they may still be linked to the same concepticon concept set, since there was no initial construction as a wordlist. So duplicates are to be expected, and this can be mentioned in the note of the file conceptlists.tsv.

laiyunfan commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-22	22	the belly	belly	1251	BELLY	
+Zhang-2019-308-23	23	the boat	boat	1844	BOAT	
+Zhang-2019-308-24	24	the body, person	body, person	1480	BODY	
+Zhang-2019-308-25	25	the body, self	body, self	1480	BODY	
```
The word in the original data is 身, originally "body", and "self" is its extended meaning.

MacyL commented:
> Yes, @MacyL, the description needs to say:
> 
> > Since the data is a collection of etymological items with very divergent meanings and elicitation glosses, we assembled the data for each language separately and added a unified layer of translation.
> 
> @MacyL, note also, that you should modify the ENGLISH column name to GLOSS, this is our standard column name for custom translations to English that we do which are not in the original data.

I have added the description to the note. About the column, I tried to change ENGLISH column name to GLOSS. However, the concepticon test complained. The error message complain about "missing source language translation english", and it's not possible to remove the "English" metadata in the json file (since the conceptlists.tsv indicates it). I checked the other concept list, they also used ENGLISH in their column instead of GLOSS. 
Any suggestion on this issue?

LinguList commented:
No, this is not true, Mei-Shin, do a `grep` on all concept lists, and check, please. 
The point is: `conceptlists.tsv` indicates that the source language is ENGLISH, but this is wrong. So please try to just leave the source language open. 

It is still possible that this will raise an error, as you have no source language then indicated. In this case, we'd just leave it with English, but indicate that these have been added by the author (Herr Lai), later, and do not appear in the original article.

Schweikhard commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-37	37	to cry (of birds or animals)	cry (of birds or animals)	1839	CRY	
+Zhang-2019-308-38	38	the dark side of the mountain	dark side of the mountain			too specific to map
+Zhang-2019-308-39	39	the deceased mother	deceased mother			too specific to map
+Zhang-2019-308-40	40	the dream	dream	2374	DREAM	
```
Are the notes "too specific to map" in the original?

Schweikhard commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-39	39	the deceased mother	deceased mother			too specific to map
+Zhang-2019-308-40	40	the dream	dream	2374	DREAM	
+Zhang-2019-308-41	41	the drop (classifier)	drop (classifier)	2087	DROP (CLASSIFIER)	
+Zhang-2019-308-42	42	be dusk, dark	dusk, dark	2297	DUSK	
```
"be dusk" sounds like this is an adjective/verb?

laiyunfan replied:
Yes... I'd unmapped it. It's not exactly dark.

Schweikhard commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-132	132	to suffer defeat	suffer defeat	3535	BE DEFEATED	
+Zhang-2019-308-133	133	the tail	tail	1220	TAIL	
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
```
Is this about the limb or only the bone?

laiyunfan replied:
In Gyalrong languages it is the limb. In Old Chinese I'm not sure, perhaps it is the bone.

Schweikhard commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-179	179	to wear	to wear	976	WEAR	
+Zhang-2019-308-180	180	to weave	to weave	133	WEAVE	
+Zhang-2019-308-181	181	the tree	tree	906	TREE	
+Zhang-2019-308-182	182	to twist, to spin	twist, spin	430	TWIST	
```
TWIST is "To wind something around something else or several things around each other." - does this describe what's meant here?

laiyunfan replied:
I'd say yes.

Schweikhard commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-183	183	two	two	1498	TWO	
+Zhang-2019-308-184	184	the udder	udder	154	UDDER	
+Zhang-2019-308-185	185	to vomit	vomit	1278	VOMIT	
+Zhang-2019-308-186	186	the wild onion	wild onion	2366	ONION	
```
ONION refers to a cultivated species of allium, and is already mapped below.

laiyunfan replied:
unmapped

Kristina-Pianykh commented on concepticondata/conceptlists/Zhang-2019-308.tsv:
```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-160	160	to itch	to itch	148	ITCH	
+Zhang-2019-308-161	161	to kill	to kill	1417	KILL	
+Zhang-2019-308-162	162	to lose	to lose	1660	LOSE	
+Zhang-2019-308-163	163	to oppose, disturb	to oppose, disturb	2883	DISTURB	
```
Can this one and the one below (Zhang-2019-308-230) be mapped to DISTURB? I can judge only by the English glosses, whose meaning is a bit broader here than just DISTURB.

laiyunfan replied:
unmapped

LinguList commented:
we can unmap it, I'd say.

AnnikaTjuka commented:
@MacyL Not sure what the status of this PR is. But we are adding more and more changes/datasets to Concepticon, which will eventually result in some conflicting files. Please keep this in mind when you continue with this PR.

LinguList commented:
Yes, this should be solved, ideally today. @MacyL, @laiyunfan, should I finalize this?

laiyunfan commented:
I think, we can finalize the PR, if @MacyL agrees.

LinguList commented:
Check: unique_concepticon_gloss
CONCEPTICON_ID      LINE_NO  ID                    NUMBER  GLOSS
----------------  ---------  ------------------  --------  -----------------------------
                          2  Zhang-2019-308-1           1  diminutive suffix
                          7  Zhang-2019-308-6           6  the Allium tuberosum
                         39  Zhang-2019-308-38         38  the dark side of the mountain
                         40  Zhang-2019-308-39         39  the deceased mother
                         43  Zhang-2019-308-42         42  be dusk, dark
                         48  Zhang-2019-308-47         47  be eldest, great
                         54  Zhang-2019-308-53         53  the farming
                         58  Zhang-2019-308-57         57  the father's sister's child
                         67  Zhang-2019-308-66         66  the fur garment
                         69  Zhang-2019-308-68         68  the grandchild, nephew
                         92  Zhang-2019-308-91         91  the millet or rice grains
                        110  Zhang-2019-308-109       109  the pliers
                        125  Zhang-2019-308-124       124  the snake brood
                        129  Zhang-2019-308-128       128  be stiff
                        135  Zhang-2019-308-134       134  there is no X such that X...
                        144  Zhang-2019-308-143       143  to carry water on back
                        147  Zhang-2019-308-146       146  to confer
                        157  Zhang-2019-308-156       156  to fall (from hand)
                        164  Zhang-2019-308-163       163  to oppose, disturb
                        168  Zhang-2019-308-167       167  to revive
                        169  Zhang-2019-308-168       168  to scoop water
                        177  Zhang-2019-308-176       176  to take care of, protect
                        187  Zhang-2019-308-186       186  the wild onion
                        218  Zhang-2019-308-217       217  the dead mother
                        220  Zhang-2019-308-219       219  be eldest, greatest
                        231  Zhang-2019-308-230       230  to oppose, disturb
                        235  Zhang-2019-308-234       234  be dusk, dark
                        244  Zhang-2019-308-243       243  to scoop water
                        257  Zhang-2019-308-256       256  be stiff
                        282  Zhang-2019-308-281       281  there is no X such that X...
                        287  Zhang-2019-308-286       286  the snake brood
                        291  Zhang-2019-308-290       290  the fur garment
                        303  Zhang-2019-308-302       302  the Allium tuberosum
                        307  Zhang-2019-308-306       306  the thighbone
                        309  Zhang-2019-308-308       308  to cry (of birds or animals)
10                       61  Zhang-2019-308-60         60  the firewood
10                      285  Zhang-2019-308-284       284  the firewood
1044                    108  Zhang-2019-308-107       107  the pillow
1044                    238  Zhang-2019-308-237       237  pillow
1173                     52  Zhang-2019-308-51         51  the eyebrow
1173                    261  Zhang-2019-308-260       260  the eyebrow
1209                      4  Zhang-2019-308-3           3  I
1209                    210  Zhang-2019-308-209       209  I
1217                     56  Zhang-2019-308-55         55  the father
1217                    253  Zhang-2019-308-252       252  the father
1220                    134  Zhang-2019-308-133       133  the tail
1220                    222  Zhang-2019-308-221       221  the tail
1231                     18  Zhang-2019-308-17         17  new
1231                    234  Zhang-2019-308-233       233  be new
1233                     99  Zhang-2019-308-98         98  the night
1233                    215  Zhang-2019-308-214       214  the night
1247                     44  Zhang-2019-308-43         43  the ear
1247                    271  Zhang-2019-308-270       270  the ear
1248                     51  Zhang-2019-308-50         50  the eye
1248                    260  Zhang-2019-308-259       259  the eye
1251                     23  Zhang-2019-308-22         22  the belly
1251                    278  Zhang-2019-308-277       277  the belly
1278                    186  Zhang-2019-308-185       185  to vomit
1278                    213  Zhang-2019-308-212       212  to vomit
1306                    128  Zhang-2019-308-127       127  the sovereign, queen
1306                    209  Zhang-2019-308-208       208  the sovereign, queen
1324                    174  Zhang-2019-308-173       173  to slip
1324                    294  Zhang-2019-308-293       293  to slip
133                     181  Zhang-2019-308-180       180  to weave
133                     267  Zhang-2019-308-266       266  to weave
1331                    119  Zhang-2019-308-118       118  the sheep
1331                    270  Zhang-2019-308-269       269  the sheep
1334                     84  Zhang-2019-308-83         83  the internal organs
1334                    276  Zhang-2019-308-275       275  the internal organs
1382                     98  Zhang-2019-308-97         97  the needle
1382                    299  Zhang-2019-308-298       298  the needle
1392                     89  Zhang-2019-308-88         88  the louse
1392                    289  Zhang-2019-308-288       288  the louse
1393                     78  Zhang-2019-308-77         77  the horn
1393                    292  Zhang-2019-308-291       291  the horn
1405                     96  Zhang-2019-308-95         95  the name
1405                    208  Zhang-2019-308-207       207  the name
1416                    173  Zhang-2019-308-172       172  to sit
1416                    214  Zhang-2019-308-213       213  to sit
1417                    162  Zhang-2019-308-161       161  to kill
1417                    241  Zhang-2019-308-240       240  to kill
1418                    153  Zhang-2019-308-152       152  to dig out (earth)
1418                    229  Zhang-2019-308-228       228  to dig out (earth)
1442                    176  Zhang-2019-308-175       175  to stand
1442                    263  Zhang-2019-308-262       262  to stand
1447                    159  Zhang-2019-308-158       158  to give
1447                    256  Zhang-2019-308-255       255  to give
148                     161  Zhang-2019-308-160       160  to itch
148                     258  Zhang-2019-308-257       257  to itch
1480                     25  Zhang-2019-308-24         24  the body, person
1480                     26  Zhang-2019-308-25         25  the body, self
1480                    295  Zhang-2019-308-294       294  the body
1480                    296  Zhang-2019-308-295       295  the body
1483                    100  Zhang-2019-308-99         99  nine
1483                    194  Zhang-2019-308-193       193  nine
1493                    102  Zhang-2019-308-101       101  one
1493                    123  Zhang-2019-308-122       122  be single
1493                    301  Zhang-2019-308-300       300  be single
1494                    152  Zhang-2019-308-151       151  to die
1494                    240  Zhang-2019-308-239       239  to die
1498                    184  Zhang-2019-308-183       183  two
1498                    196  Zhang-2019-308-195       195  two
1504                     64  Zhang-2019-308-63         63  the fly
1504                    290  Zhang-2019-308-289       289  the fly (insect)
1510                     86  Zhang-2019-308-85         85  the jaw, chin
1510                    304  Zhang-2019-308-303       303  the jaw, chin
1560                     53  Zhang-2019-308-52         52  the face
1560                    305  Zhang-2019-308-304       304  the face
1574                     63  Zhang-2019-308-62         62  to float
1574                    245  Zhang-2019-308-244       244  to float
1583                     27  Zhang-2019-308-26         26  the brain
1583                    277  Zhang-2019-308-276       276  the brain
1613                     34  Zhang-2019-308-33         33  the colour, countenance
1613                    281  Zhang-2019-308-280       280  the colour, countenance
1634                     80  Zhang-2019-308-79         79  hundred
1634                    259  Zhang-2019-308-258       258  hundred
1660                    163  Zhang-2019-308-162       162  to lose
1660                    217  Zhang-2019-308-216       216  to lose
1685                      5  Zhang-2019-308-4           4  after
1685                    226  Zhang-2019-308-225       225  after
1689                    175  Zhang-2019-308-174       174  to smoke
1689                    250  Zhang-2019-308-249       249  to smoke
1698                     13  Zhang-2019-308-12         12  to be curved
1698                    223  Zhang-2019-308-222       222  be curved
1703                    124  Zhang-2019-308-123       123  six
1703                    202  Zhang-2019-308-201       201  six
1704                    117  Zhang-2019-308-116       116  seven
1704                    190  Zhang-2019-308-189       189  seven
1705                     46  Zhang-2019-308-45         45  eight
1705                    201  Zhang-2019-308-200       200  eight
172                     122  Zhang-2019-308-121       121  the shoulder blade
172                     279  Zhang-2019-308-278       278  the shoulder blade
1767                    170  Zhang-2019-308-169       169  to scrape
1767                    204  Zhang-2019-308-203       203  to scrape
1820                      3  Zhang-2019-308-2           2  the brown bear
1820                     21  Zhang-2019-308-20         20  the bear
1820                     29  Zhang-2019-308-28         28  the brown-and-white bear
1820                    249  Zhang-2019-308-248       248  the bear
1820                    269  Zhang-2019-308-268       268  the bear
1884                     16  Zhang-2019-308-15         15  hard
1884                    132  Zhang-2019-308-131       131  be strong, hard
1884                    205  Zhang-2019-308-204       204  be hard
1886                      9  Zhang-2019-308-8           8  the armpit
1886                    273  Zhang-2019-308-272       272  the armpit
1886                    275  Zhang-2019-308-274       274  the armpit
1895                     33  Zhang-2019-308-32         32  the clothes
1895                     68  Zhang-2019-308-67         67  the garment
1906                    127  Zhang-2019-308-126       126  be sour
1906                    298  Zhang-2019-308-297       297  be sour
1913                    167  Zhang-2019-308-166       166  to press
1913                    293  Zhang-2019-308-292       292  to press
1942                     97  Zhang-2019-308-96         96  be near
1942                    297  Zhang-2019-308-296       296  be near
1991                     83  Zhang-2019-308-82         82  the intestine
1991                     85  Zhang-2019-308-84         84  the intestine
1998                     35  Zhang-2019-308-34         34  the command
1998                    207  Zhang-2019-308-206       206  to command
2014                    150  Zhang-2019-308-149       149  to descend
2014                    192  Zhang-2019-308-191       191  descend
2099                     31  Zhang-2019-308-30         30  the child
2099                    199  Zhang-2019-308-198       198  the child
212                      59  Zhang-2019-308-58         58  the field
212                     255  Zhang-2019-308-254       254  the field
2146                    158  Zhang-2019-308-157       157  to fry, roast
2146                    251  Zhang-2019-308-250       250  to fry, roast
221                      60  Zhang-2019-308-59         59  the fire
221                     248  Zhang-2019-308-247       247  the fire
2224                    141  Zhang-2019-308-140       140  to bend
2224                    148  Zhang-2019-308-147       147  to curve
2224                    237  Zhang-2019-308-236       236  to bend
2249                    137  Zhang-2019-308-136       136  be thin
2249                    284  Zhang-2019-308-283       283  be thin
2271                    179  Zhang-2019-308-178       178  to think
2271                    227  Zhang-2019-308-226       226  to think
2341                     15  Zhang-2019-308-14         14  cloudy
2341                    300  Zhang-2019-308-299       299  be dark, cloudy
2345                    113  Zhang-2019-308-112       112  the rabbit, hare
2345                    200  Zhang-2019-308-199       199  the rabbit, hare
2350                     45  Zhang-2019-308-44         44  the earthworm
2350                    288  Zhang-2019-308-287       287  the earthworm
2374                     41  Zhang-2019-308-40         40  the dream
2374                    216  Zhang-2019-308-215       215  the dream
243                      65  Zhang-2019-308-64         64  be fresh
243                     308  Zhang-2019-308-307       307  be fresh
321                     146  Zhang-2019-308-145       145  to chew
321                     211  Zhang-2019-308-210       210  to chew
3326                     75  Zhang-2019-308-74         74  the hide
3326                    302  Zhang-2019-308-301       301  the hide
353                     130  Zhang-2019-308-129       129  to strike
353                     230  Zhang-2019-308-229       229  to strike
3538                    178  Zhang-2019-308-177       177  to take off
3538                    274  Zhang-2019-308-273       273  to take off
362                     131  Zhang-2019-308-130       130  the string, cord
362                     268  Zhang-2019-308-267       267  the string, cord
3684                    165  Zhang-2019-308-164       164  to peck
3684                    212  Zhang-2019-308-211       211  to peck
3764                     49  Zhang-2019-308-48         48  be evil
3764                    228  Zhang-2019-308-227       227  be evil
425                      88  Zhang-2019-308-87         87  be late
425                     236  Zhang-2019-308-235       235  be late
430                     183  Zhang-2019-308-182       182  to twist, to spin
430                     266  Zhang-2019-308-265       265  to twist, to spin
492                     139  Zhang-2019-308-138       138  three
492                     191  Zhang-2019-308-190       190  three
493                      62  Zhang-2019-308-61         61  five
493                     197  Zhang-2019-308-196       196  five
598                     120  Zhang-2019-308-119       119  the shell
598                     242  Zhang-2019-308-241       241  the shell
615                      79  Zhang-2019-308-78         78  the horse
615                     306  Zhang-2019-308-305       305  the horse
617                      82  Zhang-2019-308-81         81  the ice
617                     203  Zhang-2019-308-202       202  the ice
655                     111  Zhang-2019-308-110       110  the poison
655                     243  Zhang-2019-308-242       242  the poison
670                     109  Zhang-2019-308-108       108  the plant root
670                     206  Zhang-2019-308-205       205  the root
675                      50  Zhang-2019-308-49         49  the excrement
675                     224  Zhang-2019-308-223       223  the excrement
677                      11  Zhang-2019-308-10         10  the axe
677                     233  Zhang-2019-308-232       232  the axe
681                     151  Zhang-2019-308-150       150  to destroy
681                     246  Zhang-2019-308-245       245  to destroy
712                     114  Zhang-2019-308-113       113  be rich
712                     221  Zhang-2019-308-220       220  be rich
72                       32  Zhang-2019-308-31         31  the claw
72                      252  Zhang-2019-308-251       251  the claw
732                     189  Zhang-2019-308-188       188  your
732                     193  Zhang-2019-308-192       192  your
735                     104  Zhang-2019-308-103       103  the owl
735                     239  Zhang-2019-308-238       238  the owl
771                      72  Zhang-2019-308-71         71  the hat
771                     225  Zhang-2019-308-224       224  the hat
782                     149  Zhang-2019-308-148       148  to defeat
782                     232  Zhang-2019-308-231       231  to defeat
808                     154  Zhang-2019-308-153       153  to drop, drop
808                     247  Zhang-2019-308-246       246  to drop, drop
832                     112  Zhang-2019-308-111       111  the pulse, beans
832                     283  Zhang-2019-308-282       282  the pulse, beans
977                      10  Zhang-2019-308-9           9  the arrow
977                     262  Zhang-2019-308-261       261  the arrow
981                      47  Zhang-2019-308-46         46  the elbow
981                     272  Zhang-2019-308-271       271  the elbow
999                      87  Zhang-2019-308-86         86  the joint
999                     264  Zhang-2019-308-263       263  the joint
Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK
Check: valid_concepticon_id
OK

LinguList commented:
I just resolved the conflicts, and I'll merge it now.
